engines/dragons/cutscene.cpp

```diff
@@ -0,0 +1,888 @@
+																									DAT_80072df0->startWalk(0xe8,0xa8,2);
+																									while (DAT_80072df0->isFlagSet(ACTOR_FLAG_10)) {
+																										_vm->waitForFrames(1);
+																									}
+																									DAT_80072de8->startWalk(0x97,0x37,2);
+																									DAT_80072dec->startWalk(0x97,0x37,2);
+																									DAT_80072df4->startWalk(0x97,0x37,2);
+																									DAT_80072df0->waitUntilFlag8SetThenSet1000AndWaitFor4();
+																									DAT_80072df0->updateSequence(6);
+																									uint16 dialog[2000];
+																									dialog[0] = 0;
+																									_vm->_talk->loadText(0x5ea2, dialog, 2000);
+
+																									_vm->_talk->displayDialogAroundPoint(dialog,0x27,0xc,0xc01,0,0x5ea2);
+																									DAT_80072df0->waitUntilFlag8And4AreSet();
+																									DAT_80072df0->x_pos = 0xcf;
+																									DAT_80072df0->y_pos = 0x90;
+																									DAT_80072df0->startWalk(0x97, 0x37, 2);
+																									DAT_80072df0->updateSequence(7);
+//TODO
+//																									if (((DAT_8008e7e8 != 0) || (DAT_8008e848 != 0))
+//																										|| ((DAT_8008e844 != 0 || (DAT_8008e874 != 0)))
+//																											) {
+//																										FUN_8001a7c4((uint)DAT_8008e7e8,
+//																													 (uint)DAT_8008e844,
+//																													 (uint)DAT_8008e848,
+//																													 (uint)DAT_8008e874);
+//																									}
+
+																									dialog[0] = 0;
+																									_vm->_talk->loadText(0x5ecc, dialog, 2000);
+																									_vm->_talk->displayDialogAroundPoint(dialog,0x14,6,0xc01,0,0x5ecc);
+																									_vm->waitForFrames(0x3c);
+																								}
+																							}
+																						}
+																					}
+																				}
+																			}
+																		}
+																	}
+																}
+															}
+														}
+													}
+												}
+											}
+										}
+									}
+								}
+							}
+						}
+					}
+				}
+			}
+		}
+	}
+	// fade_related_calls_with_1f();
+	_vm->clearFlags(ENGINE_FLAG_20000);
+	// DisableVSyncEvent();
+	//file_read_to_buffer(s_cursor.act_80011c44,actor_dictionary);
+	// EnableVSyncEvent();
+	if (isFlag10Set) {
+		_vm->setFlags(ENGINE_FLAG_10);
+	} else {
+		_vm->clearFlags(ENGINE_FLAG_10);
+	}
+
+	_vm->_dragonINIResource->setFlickerRecord(flicker);
+	cursorInventoryClearFlag400();
+	_vm->clearUnkFlags(ENGINE_UNK1_FLAG_2);
+
+}
+
+//fadeScreenAndResetActor
+void CutScene::fadeScreenAndResetActor(Actor *actor) {
+	// TODO fade_related_calls_with_1f();
+	//DisableVSyncEvent();
+	actor->reset_maybe();
+	//EnableVSyncEvent();
+
+}
+
+void CutScene::closeUpShotOnActor(uint16 resourceId, uint16 sequenceId, int16 x, uint32 param_4) {
+	FUN_8003d8e8(resourceId, sequenceId, x, param_4);
```

Comment:
All capitals reserved for the constants according to our code formatting conventions. Please, rename